app/src/main/java/com/github/fribourgsdp/radio/util/MyTextToSpeech.kt

```diff
@@ -0,0 +1,75 @@
+package com.github.fribourgsdp.radio.util
+
+import android.content.Context
+import android.speech.tts.TextToSpeech
+import android.widget.Toast
+import com.github.fribourgsdp.radio.R
+import com.github.fribourgsdp.radio.config.language.LanguageManager
+import com.github.fribourgsdp.radio.external.musixmatch.MusixmatchLyricsGetter
+import java.util.*
+
+
+const val TTS_INITIALIZATION_RETRY_DELAY_MS = 1000
+
+open class MyTextToSpeech(private val applicationContext: Context) : TextToSpeech.OnInitListener {
+    private var tts: TextToSpeech? = null
```

Comment:
I saw that you force the call to this `tts` every time, which isn't that safe. So either use the safe call `?.` or put this as non-nullable.

Reply:
Ok, going to change this.